publish/dir.targets

```diff
@@ -4,7 +4,7 @@
 
   <Target Name="GatherPublishToAzureBinaries">
     <PropertyGroup>
-      <RelativeBlobPath>$(Channel)/Binaries/$(SharedFrameworkNugetVersion)</RelativeBlobPath>
+      <RelativeBlobPath>Runtime/$(SharedFrameworkNugetVersion)</RelativeBlobPath>
```

Comment:
FYI - this will conflict with @karajas's change since he is moving these properties around.